src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Color ballColor = colorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100);
```

Comment:
All magic numbers in your code should be constants. The number 100 should be extracted to a constant field with a meaningful name.